en/user-guide/layerexplanations/namingconventions.rst

```diff
@@ -0,0 +1,49 @@
+Naming conventions
+==================
+
+Schemas in the QGEP-Database
+---------------------------
+The QGEP-Database has six qgep-schemas:
+
+* qgep_od: in this schema you find the tables, that correspond to the VSA-DSS-datamodel (cover, wastewaternode, channel etc.) Here are the tables you normally work.
+
+* qgep_vl: is the schema for valuelists. There is a table for every valuelist of the VSA-DSS. As a QGEP-User you can not edit this tables.
+
+* qgep_network: schema for the tables that are nesseccary to use the network-topology with the qgep-tools. You can rebuild this tables from the QGEP-tools.
+
+* qgep_sys: special tables for administration of the database. As a QGEP-User you can not edit this tables.
+
+* qgep_import and qgep_swmm: schema for additional plugins/functions (Qfield and working with SWMM)
+
+
+Layer names
+-----------
+
+* Layer names have often a prefix like od_ or vl_.
+This showes you, in which schema to look for the table. od_organisation is the the table organisation in the qgep_od-schema.
+
+* Layer Names that begin with vw_
+Two or more VSA-DSS-Tables are joined in a view (vw). The layer has the name of the main-table. Example: vw_qgep_reach: the main layer is the table reach (not the table channel).
+It is important to know the main-table to be sure, that you use the correct attributes. 
+
+* Layer vw_network_
+This are views in the schema qgep_network
+
+Attibut names
+-------------
+
+* Prefixes in attribut names
+In a view, you find fields from different tables. In easier cases, there is only a subclass and a superclass is included. In the qgep-mainlayers there are serveral sub- und superclasses.
+With the prefix of the attribut name, you know, to which table an attribut belongs. The prefix are two letters that belong to a table of the VSA-DSS-datamodel. Example: co = cover, ws = wastewater structure.
```

Comment:
attribute in English instead of attribut

Reply:
deutsch Attribute (Mehrzahl) wird dann zu attributes (in English)

Reply:
```suggestion
With the prefix of the attribute name, you know, to which table an attribute belongs. The prefix are two letters that belong to a table of the VSA-DSS datamodel. Example: co = cover, ws = wastewater structure.
```